terraform/modules/apps/signon/variables.tf

```diff
@@ -0,0 +1,62 @@
+variable "vpc_id" {
+  type = string
+}
+
+variable "cluster_id" {
+  description = "ECS cluster to deploy into."
+  type        = string
+}
+
+variable "task_role_arn" {
+  description = "ARN of IAM role for app's container (ECS task) to talk to other AWS services."
+  type        = string
+}
+
+variable "execution_role_arn" {
+  description = "ARN of IAM role for the ECS container agent and Docker daemon to manage the app container."
+  type        = string
+}
+
+variable "service_name" {
+  description = "Service name of the Fargate service, cluster, task etc."
+  type        = string
+  default     = "signon"
+}
+
+variable "private_subnets" {
+  description = "Subnet IDs to use for non-Internet-facing resources."
+  type        = list
+}
+
+variable "govuk_management_access_sg_id" {
+  description = "Group used to allow access by management systems"
+  type        = string
+  default     = "sg-0b873470482f6232d"
```

Comment:
This default shouldn't be here as it's specific to the test environment. It's already being passed down from the top-level module but we haven't cleaned up some of the other occurrences of it yet (sorry about that).
```suggestion
```

Reply:
Please remove this default.